index.js

```diff
@@ -0,0 +1,23 @@
+import './style.css';
+import {
+  display, add, modify, removeItem,
+} from './functions.js';
+import { update } from './updates.js';
+
+const TodoItemsV2 = JSON.parse(localStorage.getItem('ToDoItems')) || [];
+if (TodoItemsV2.length > 0) {
+  for (let i = 0; i < TodoItemsV2.length; i += 1) {
+    display(i);
+    modify(i);
+  }
+  removeItem();
+  update();
+}
+
+const input = document.getElementById('add');
+const submit = document.getElementById('submit');
```

Comment:
Try to move the global variables declaration to the top of the document just after the imports statements, in that way we can directly see from the beginning which variables you will be using in your modal!  like this you can see and have an overview.
![image](https://user-images.githubusercontent.com/105079888/184727044-dd76bf62-ea7a-4618-9ce5-e0ca40fa0c93.png)